core/src/main/java/org/arquillian/smart/testing/configuration/ConfigurationLoader.java

```diff
@@ -22,36 +23,60 @@
 
     public static final String SMART_TESTING_YML = "smart-testing.yml";
     public static final String SMART_TESTING_YAML = "smart-testing.yaml";
+    public static final String SMART_TESTING_CONFIG = "smart.testing.config";
+
     private static final Logger logger = Log.getLogger();
 
     public static Configuration load() {
         return load(Paths.get("").toAbsolutePath().toFile());
     }
 
     public static Configuration load(File projectDir) {
+        Map<String, Object> yamlConfiguration;
+
+        final String customConfigFile = System.getProperty(SMART_TESTING_CONFIG);
+        if (customConfigFile != null) {
+            yamlConfiguration = getConfigurationFromCustomConfigFile(customConfigFile);
+        } else {
+            yamlConfiguration = getConfigurationFromDefaultConfigFile(projectDir);
```

Comment:
This as well might be a one liner:
`yamlConfiguration = readConfiguration(System.getProperty(SMART_TESTING_CONFIG, projectDir));`

Assuming we can have just one method taking care of loading a file from a given location.